app/jobs/endpoint_check_job.rb

```diff
@@ -0,0 +1,36 @@
+# Confirms existence of PreservedCopy on an endpoint
+# Notify ResultsRecorder, with success or failure of existence.
+class EndpointCheckJob < ApplicationJob
+  queue_as :endpoint_check
```

Comment:
In practice we will need to separate queues per AWS region (like the delivery workers).  We can represent that at the model level w/ different classes (like delivery workers), or we can just invoke this job like:
```ruby
EndpointCheckJob.set(queue: :endpoint_check_us_west_2).perform_later(pc)
EndpointCheckJob.set(queue: :endpoint_check_us_east_1).perform_later(pc)
...
```
The code doesn't need to change here, but it probably is good to mention that this queue name itself is never expected to be used since it affects how we spin up resque workers.

Reply:
Cool. i can add a follow on commit or ticket the addition of r the AWS region queues.